packages/augur-ui/src/modules/markets-list/components/landing-hero.tsx

```diff
@@ -1,38 +1,89 @@
 import React from 'react';
 
 import Styles from 'modules/markets-list/components/landing-hero.styles.less';
-import { PrimaryButton, ExternalLinkButton } from 'modules/common/buttons';
-import { MODAL_SIGNUP } from 'modules/common/constants';
+import { ExternalLinkButton, PrimaryButton } from 'modules/common/buttons';
+import { MODAL_SIGNUP, THEMES } from 'modules/common/constants';
 import { useAppStatusStore } from 'modules/app/store/app-status';
+import { Slider } from 'modules/markets-list/components/slider';
+import Image from '../../../assets/images/landing-slider-img.png';
+import classNames from 'classnames';
+
+const images = [{
+  alignment: 'center',
+  image: Image,
+  text: 'No Limits. Lower Fees.\nBet on anything you want.',
+  button: {
+    text: 'Sign up to start betting',
+    link: 'https://google.com/'
+  }
+}, {
+  alignment: 'left',
+  image: Image,
+  text: 'No Limits. Lower Fees.\nBet on anything you want. 2',
+  button: {
+    text: 'Sign up to start betting',
+    link: 'https://google.com/'
+  }
+}, {
+  alignment: 'right',
+  image: Image,
+  text: 'No Limits. Lower Fees.\nBet on anything you want. 3',
+  button: {
+    text: 'Sign up to start betting',
+    link: 'https://google.com/'
+  }
+}];
 
 export const LandingHero = () => {
   const {
     actions: { setModal },
   } = useAppStatusStore();
+  const { theme } = useAppStatusStore();
```

Comment:
put this above, we don't call `useAppStatusStore()` more than once.

Reply:
Can't believe I didn't see that! I gotta get my eyes checked.